### PR TITLE
Add agama test for leap for auto-installation

### PR DESCRIPTION
**Pull request by lemon-suse**
- Related ticket: https://progress.opensuse.org/issues/167087
- Needles: N/A
- Verification run: https://openqa.opensuse.org/tests/overview?version=agama-9.0&distri=opensuse&build=lemon-suse%2Fos-autoinst-distri-opensuse%23add-agama-test-for-leap

### CODE REVIEW

File: test_data/yam/agama_auto_leap.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+os_release_name: Leap
+guided_partitioning:
+  disks:
+    - vdb
+unused_disks:
+  - vda
```
Comment (jknphy) [inline test_data/yam/agama_auto_leap.yaml:7]: we are not doing any partitioning to have test data to validate that.

File: lib/Yam/Agama/Pom/AgamaUpAndRunningPage.pm [hunk 2]
```diff
@@ -16,7 +16,7 @@ use testapi;
 sub new {
     my ($class, $args) = @_;
     return bless {
-        tag_array_ref_any_first_screen_shown => [qw(agama-product-selection agama-configuring-the-product)]
+        tag_array_ref_any_first_screen_shown => [qw(agama-product-selection agama-configuring-the-product agama-prepared-disks)]
```
Comment (jknphy) [inline lib/Yam/Agama/Pom/AgamaUpAndRunningPage.pm:19]: why we need this?

File: test_data/yam/agama_auto_leap.yaml (hunk 1 above)
Comment (lemon-suse) [reply test_data/yam/agama_auto_leap.yaml]: Thanks, removed.

File: lib/Yam/Agama/Pom/AgamaUpAndRunningPage.pm (hunk 2 above)
Comment (lemon-suse) [reply lib/Yam/Agama/Pom/AgamaUpAndRunningPage.pm]: Updated, I'm running a VR https://openqa.opensuse.org/tests/4503871#live

File: test_data/yam/agama_auto_leap.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (jknphy) [inline test_data/yam/agama_auto_leap.yaml:1]: why to have two files exactly the same?

**Pull request merged**